webapp/src/component/App.tsx

```diff
@@ -85,17 +89,22 @@ const MandatoryDataProvider = (props: any) => {
       import('@openreplay/tracker').then(({ default: Tracker }) => {
         window.openReplayTracker = new Tracker({
           projectKey: openReplayApiKey,
+          __DISABLE_SECURE_MODE: true,
         });
         window.openReplayTracker.start();
       });
     }
-  }, [config?.clientSentryDsn]);
+    setOpenReplayTracker(window.openReplayTracker);
+  }, [config?.clientSentryDsn, config?.openReplayApiKey]);
 
   useEffect(() => {
-    if (userData && window.openReplayTracker) {
-      window.openReplayTracker.setUserID(userData.username);
+    if (userData && openReplayTracker) {
+      openReplayTracker.setUserID(userData.username);
+      setTimeout(() => {
+        openReplayTracker?.setUserID(userData.username);
```

Comment:
This is to make sure? 😄

Reply:
He said, I should do it twice. 😄  But I don't think that calling it 2 times right after should be enough so I did it this way. He said, they'll fix it.